.github/workflows/reusable_build.yml

```diff
@@ -98,6 +98,8 @@ jobs:
                   tree_version_new=$(cargo +$RUST_NIGHTLY tree --depth 1 --prefix none | grep $crate | awk '{print $2}' | tr -d 'v')
                   if [[ $tree_version_curr != $tree_version_new ]]; then
                       echo "$crate : version is $tree_version_curr whereas $tree_version_new is available; please update !"
+                      echo "Please activate dependabot to automatically update the Rust SDK dependencies"
```

Comment:
Instead of `Please activate dependabot`

`Don't forget to activate dependabot in your repository settings `